### ISSUE TITLE
How do I fine-tune hyperparameters for a model from Huggingface library

### ISSUE THREAD

**Issue by thak123** [huggingface/transformers#4895]
# ❓ Questions & Help

## Details
Hi, I am new to Hugging-face library and want to fine tune hyper-parameters of the mBert model. I have a simple classification head on top the cls token. I am getting around 76% accuracy. 
Is there any read-me or notebook available for doing the same.

**Comment by tienduccao** [comment]
You can take a look at this blog post https://mccormickml.com/2019/07/22/BERT-fine-tuning/

**Comment by thak123** [comment]
Thank You
